### ISSUE TITLE
Search control OnSearch does not exit loading state of button

### ISSUE THREAD

**Issue by JosefRousal** [ant-design-blazor/ant-design-blazor#1183]
### Describe the bug

When using the OnSearch handler of the Search component the function passed as a parameter is called and completes but the loading state of the search button does not go away, preventing further searches.

### Steps to reproduce (please include code)

The example on the demo website is also broken in the same way, as an example.

```
<div>
    <Search Placeholder="input search text" Size="@InputSize.Large" EnterButton="@("Search")" @bind-Value="@txtValue" OnSearch="OnSearch"/>
</div>

@inject MessageService message;

@code{

    private string txtValue { get; set; }

    private void Handle(string value)
    {
        Console.WriteLine(value);
    }

    public async Task OnSearch()
    {
        await message.Loading($"searching {txtValue}", 2);
    }

}

```

### Further technical details
- AntDesign Nuget Package version 0.7.0-nightly-2102281452

**Comment by JosefRousal** [comment]
To be clear, this won't be fixed until the 0.8.0 release?

**Comment by ElderJames** [comment]
Thanks for contacting us @JosefRousal .

As planned, we are about to release 0.7 and adjust the release cycle. And the nightly releases will be made to allow users to use the latest changes.
The remaining issues will be released next week.
